Microsoft.Toolkit.Uwp.UI.Controls/Expander/Expander.xaml

```diff
@@ -0,0 +1,342 @@
+                                          ContentTransitions="{TemplateBinding ContentTransitions}"
+                                          Content="{TemplateBinding Content}"
+                                          HorizontalContentAlignment="{TemplateBinding HorizontalContentAlignment}"
+                                          Padding="{TemplateBinding Padding}"
+                                          VerticalContentAlignment="{TemplateBinding VerticalContentAlignment}"
+                                          HorizontalAlignment="Left"
+                                          RenderTransformOrigin="0.5,0.5" />
+                    </Grid>
+                </ControlTemplate>
+            </Setter.Value>
+        </Setter>
+    </Style>
+
+    <Style TargetType="controls:Expander">
+        <Setter Property="Header" Value="Header" />
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="controls:Expander">
+                    <Grid>
+                        <VisualStateManager.VisualStateGroups>
+                            <VisualStateGroup x:Name="ExpandedStates">
+                                <VisualState x:Name="Expanded">
+                                    <VisualState.Setters>
+                                        <Setter Target="PART_MainContentRow.Height" Value="Auto" />
+                                    </VisualState.Setters>
+                                </VisualState>
+                                <VisualState x:Name="Collapsed">
+                                    <VisualState.Setters>
+                                        <Setter Target="PART_MainContentRow.Height" Value="0" />
+                                    </VisualState.Setters>
+                                </VisualState>
+                            </VisualStateGroup>
+                        </VisualStateManager.VisualStateGroups>
+                        
+                        <Grid>
+                            <Grid.RowDefinitions>
+                                <RowDefinition Height="Auto" />
+                                <RowDefinition Height="Auto" x:Name="PART_MainContentRow" />
+                            </Grid.RowDefinitions>
+                            
+                            <ToggleButton x:Name="PART_ExpanderToggleButton" Height="40"
+                                          Style="{StaticResource HeaderToggleButtonStyle}" 
+                                          VerticalAlignment="Top" HorizontalAlignment="Stretch"
+                                          ContentTemplate="{TemplateBinding HeaderTemplate}" Content="{TemplateBinding Header}"
+                                          IsChecked="{Binding IsExpanded, Mode=TwoWay, RelativeSource={RelativeSource TemplatedParent}}" />
+                            
+                            <ContentPresenter Grid.Row="1" 
```

Comment:
Instead of changing the row height, default the visibility to Collapsed. In the Expanded state set it to Visible and make the Collapsed state empty. In OnApplyTemplate Check for IsExpanded. The 90% use case is that the content will be collapsed